doc/configuration.rst

```diff
@@ -797,6 +803,57 @@ Such device could be a signal generator.
 Used by:
   - `PyVISADriver`_
 
+Providers
+~~~~~~~~~
+Providers describe directories that are accessible by the target over a
+specific protocol.
+This is useful for software installation in the bootloader (via TFTP) or
+downloading update artifacts under Linux (via HTTP).
+
+They are used with the ManagedFile helper, which ensures that the file is
+available on the server and then creates a symlink from the internal directory
+to the uploaded file.
+The path for the target is generated by replacing the internal prefix with the
+external prefix.
+
+For now, the TFTP/NFS/HTTP server needs to be configured before using it from
+labgrid.
+
+.. _TFTPProvider:
+.. _NFSProvider:
+.. _HTTPProvider:
+
+TFTPProvider / NFSProvider / HTTPProvider
++++++++++++++++++++++++++++++++++++++++++
+
+.. code-block:: yaml
+
+   TFTPProvider:
+     internal: "/srv/tftp/board-23/"
+     external: "board-23/"
```

Comment:
Another example with the `HTTPProvider` is required here to understand that the URL should be part of the external string.

Reply:
I've added a HTTPProvider example.